internal/cephfs/store/volumeoptions.go

```diff
@@ -151,7 +151,7 @@ func validateMounter(m string) error {
 	return nil
 }
 
-func extractMounter(dest *string, options map[string]string) error {
+func ExtractMounter(dest *string, options map[string]string) error {
```

Comment:
Instead of exporting `ExtractMounter()`, can you clean it up and have this as a function of `VolumeOptions`?

In `NodePublish()`, you can then write something like this:
```go
        volOptions := &store.VolumeOptions{}
	defer volOptions.Destroy()

	if err := volOptions.DetectMounter(req.GetVolumeContext()) {
```